src/redux/features/admin/activateUserAcountSlice.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable prettier/prettier */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import API from '@/api/api';
+import isAuth from '@/helpers/isAuth';
+
+export const activateUserAcount = createAsyncThunk(
+  'activate/user',
+  async (data: string) => {
+    const body={
+      email:data,
+    }
+    const { data: activateUser } = await API.patch('/users/activate',body, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    return activateUser;
+  },
+);
+
+interface InitialState {
+  user: any | null;
```

Comment:
Using `any `type should generally be avoided because it bypasses type checking and can lead to potential type-related issues.

Please use the defined `User` type here. 
hint: `user: User | null;`